src/lib.rs

```diff
@@ -28,6 +28,15 @@
 //! }
 //! ```
 //!
+//! # Version format
+//!
+//! Versions can be expressed in the following formats
+//!
+//!   * "1.2" or ">= 1.2": At least version 1.2
+//!   * ">= 1.2, < 2.0": At least version 1.2 but less than version 2.0
+//!
+//! In the future more complicated version expressions might be supported.
```

Comment:
Maybe add a comment explaining it's different from the semver syntax used to define classic deps, so it's clear for users. It's a bit confusing to have multiple syntax in the same file so best to be explicit I think.

Reply:
Good idea